PhysicsTools/PatAlgos/plugins/PATJetSelector.h

```diff
@@ -21,15 +21,14 @@
 
 namespace pat {
 
-  class PATJetSelector : public edm::EDFilter {
+  class PATJetSelector : public edm::global::EDFilter<> {
```

Comment:
Can't be global since it uses the StringCutObjectSelector which is not const thread-safe.